documentation/source/simulator_support.rst

```diff
@@ -256,6 +256,35 @@ Support is preliminary.
 Noteworthy is that despite GHDL being a VHDL simulator, it implements the VPI interface.
 
 
+.. _sim-ghdl-waveforms:
+
+Waveforms
+---------
+
+To get waveforms in VCD format, set the :make:var:`SIM_ARGS` option to ``--vcd=anyname.vcd``,
+for example in a Makefile:
+
+  .. code-block:: make
+
+    SIM_ARGS+=--vcd=anyname.vcd
+
+The option can be set on the command line, as shown in the following example.
+
+  .. code-block:: bash
+
+    make SIM=ghdl SIM_ARGS=--vcd=anyname.vcd
+
+    # or define the SIM_ARGS environment variable in bash
+
+    export SIM_ARGS=--vcd=anyname.vhd; make SIM=ghdl
+
+    # or in csh
+
+    setenv SIM_ARGS=--vcd=anyname.vhd; make SIM=ghdl
```

Comment:
I want to get some other opinions on this. It does work and it's convenient, but I feel like recommending this will lead to people forgetting or not realizing it's set and it causing them issues. So I'd rather not mention it.

Reply:
I accepted @eric-wieser suggestion of `SIM_ARGS=--vcd=anyname.vhd make SIM=ghdl`